### PR TITLE
add win/arm64 factory classes and build worker

### PR DESCRIPTION
**Pull request by nsait-linaro**
The change adds factory classes for windows/arm64 platforms and adds build worker `linaro-win-arm64`

### CODE REVIEW

File: master/custom/builders.py [hunk 1]
```diff
@@ -185,6 +187,10 @@ def get_builders(settings):
         ("AMD64 Cygwin64 on Windows 10", "bray-win10-cygwin-amd64", UnixBuild, UNSTABLE),
         ("PPC64 AIX", "edelsohn-aix-ppc64", AIXBuild, UNSTABLE),
         ("PPC64 AIX XLC", "edelsohn-aix-ppc64", AIXBuildWithXLC, UNSTABLE),
+
+        # Windows/arm64
+        ("ARM64 Windows", "linaro-win-arm64", WindowsARM64Build, STABLE),
+        ("ARM64 Windows Non-Debug", "linaro-win-arm64", WindowsARM64ReleaseBuild, STABLE),
```
Comment (vstinner) [inline master/custom/builders.py:193]: Please start with UNSTABLE. I prefer to wait until it is reliable before marking them as STABLE.

Comment (nsait-linaro) [reply master/custom/builders.py]: Sure. I've added a new patch to mark the workers as UNSTABLE

**Comment by vstinner** [comment]
Merged. The config will be deployed in the next cron task (in 1h? I'm not sure).

**Comment by zooba** [comment]
Thanks, Victor!

**Comment by vstinner** [comment]
ARM64 Windows 3.x first green build: https://buildbot.python.org/all/#/builders/729/builds/2

ARM64 Windows Non-Debug 3.x orange build (success, test_asyncio random failure, it's a known issue): https://buildbot.python.org/all/#/builders/730/builds/3

**Comment by vstinner** [comment]
@pablogsal is the Python 3.10 and 3.11 release manager. I prefer to delegate the decision to him :-)

**Comment by pablogsal** [comment]
> win/arm64 buildbot worker has been up for nearly 10 days now. Except for the known intermittent issue for test_asyncio rest looks fine to me. @vstinner Is it a good time to mark the buildbot worker as stable?

I am fine marking it as stable, can you open a PR for it?

**Pull request merged**